.github/ISSUE_TEMPLATE/add-new-platform.md

```diff
@@ -0,0 +1,42 @@
+# Adding a new supported platform(N)
+
+## At Development
+
+- [ ] Create a WIP draft pull request on CoreOS-Assembler with new platform support [here](https://github.com/coreos/coreos-assembler)
+- [ ] Create a WIP draft pull request on the [fedora-web](https://pagure.io/fedora-web/websites) project
+    - [ ] `Add platform to the sites/static/js/coreos-download.js`
+- [ ] Create a WIP draft pull request on the [stream-metadata-go](https://github.com/coreos/stream-metadata-go)
+    - [ ] `Add platform to the Media struct in release/release.go`
+    - [ ] `Add supporting code for new platform to _toStreamArch_ func found in the release/translate.go` 
+- [ ] Create a WIP draft pull request on [build pipeline](https://github.com/coreos/fedora-coreos-pipeline)
+    - [ ] `Add platform to the list found in jobs/build.Jenkinsfile for building the new artifact`
+- [ ] Create a Wip draft pull request on [fedora-coreos-browser](https://github.com/coreos/fedora-coreos-browser to update the release [browser](https://builds.coreos.fedoraproject.org/browser)
+    - [ ] `Add a list element for new platform in browser/index.html`
+- [ ] Create a WIP draft pull request for documentation for provisioning new platform [here](https://github.com/coreos/fedora-coreos-docs)
+    - [ ] `Add a provisioning-[platform].adoc that walks through how to setup the new platform` 
+
+## At Release
+
+1. Merge:
+- [ ] Platform support enhancements made it to CoreOS-Assembler
+- [ ] Fedora website changes
+- [ ] Stream-metadata-go changes
+- [ ] Fedora-coreos-browser changes
+2. Wait for updates made to CoreOS-Assembler to be propagated to latest container
+    - [ ] Download Latest version of CoreOS-Assembler container. Verify platform support functionality. 
+3. Merge:
+- [ ] Build pipeline changes
+4. Tag for bot pickup: [see semantic versioning](https://semver.org)
+- [ ] Long tag stream-metadata-go commit with an appropriate new version.
+- [ ] Long tag fedora-coreos-stream-generator commit with a new appropriate version.
+    ```
+    //Ensure gpg key for signing in github settings that is associated to redhat email.
+    git status
+    //Verify you are on the upstream repo's main branch
+    git tag -s v#.#.#
+    //replace # with numbers
+    //give appropriate detail to tag, check pervious tags with 'git show v#.#.#'
+    git push git@github.com:coreos/targeted-repo.git v#.#.#
+    ```
+5. Wait for FCOS release cycle, and then merge documentation
```

Comment:
Specifically, wait for the new images to reach `stable`.  In the future that might take multiple cycles.

Reply:
Much better.